five-minute-serverless-backend/README.md

```diff
@@ -0,0 +1,209 @@
+# Five Minute Serverless Backend
+*How to build a **serverless backend** for your web applciation and have it up in running on the cloud in minutes.*
+
+This tutorial takes you through 5 steps to building a backend for your web application:
+1. Setup your Binaris environment
+1. Create a serverless function for your backend code
+1. Update your function to work from a browser
+1. Call your function from frontend code
+
+## Setup your Binaris environment
+
+If you do not have a Binaris account, [sign up for free](https://binaris.com/signup?t=8CDa37). At the end of the sign-up process you will receive an account ID and API key. Save them for the next steps.
+
+Install the CLI:
+
+```
+$ npm install binaris -g
+```
+
+Run `bn login` and paste in the API key received above to authenticate your account.
+
+## Create a serverless function for your backend code
+
+Create a new folder and use the CLI to create the template code for your function:
+
+```
+$ mkdir backend
+$ cd backend
+$ bn create node8 public_backend
+Created function public_backend in .../backend
+  (use "bn deploy public_backend" to deploy the function)
+  ```
+
+This will create a configuration file `binaris.yml` and source file `function.js`.
+
+For the sake of this tutorial, we'll build the simplest possible backend to convert strings to upper case. Let's edit the source code for our function:
+
+```diff
+> function.js
+---
+ exports.handler = async (body, context) => {
+-  const name = context.request.query.name || body.name || 'World';
+-  return `Hello ${name}!`;
++  return (typeof(body) === 'string' ? body : '').toUpperCase();
+ }
+```
+
+We can now deploy our function to the cloud:
+
+```
+$ bn deploy public_backend
+Deployed function public_backend
+Invoke with one of:
+  "bn invoke public_backend"
+  "curl https://run.binaris.com/v2/run/<account-id>/public_backend"
```

Comment:
I still think consistency is good. If you feel `<account-id>` is easier to grasp, and since it matches `accountId` used elsewhere, please update`<Your_Account_Number>` on https://github.com/binaris/devsite as well.

Reply:
I think <account-id> is clearer, as we use ID everywhere and not name. Not sure what exactly you want me to do here.